### PR TITLE
[linux] fix ubuntu packaging

### PR DESCRIPTION
**Pull request by wsnipex**
only ubuntu 20.04 is supported

@Rechi this is will work for ubuntu 20.04, are my jenkinsfile changes correct?

### CODE REVIEW

**Comment by phunkyfish** [comment]
A question. So this will build the addon and allow CI to pass. But unless the configuration is identical to the what we use it won't function the same as other platforms correct?

**Comment by wsnipex** [comment]
which configuration? It's correct that it won't build against the same ffmpeg version as kodi.
If you absolutely need this, you should probably use kodi's ffmpeg via some addon API.

**Comment by phunkyfish** [comment]
By the configuration within depends on the addon source by which ffmpeg is built. It is not the same as kodi’s config.

I guess I can just add a note to the README letting any Debian-esc users to build the addon manually if they want the full feature set.

**Comment by wsnipex** [comment]
yes, correct. But ubuntu's ffmpeg is built with many more options then yours here.
I don't see any you specify that should not be available, but only testing can reveal possible issues

**Comment by phunkyfish** [comment]
> yes, correct. But ubuntu's ffmpeg is built with many more options then yours here.
> I don't see any you specify that should not be available, but only testing can reveal possible issues

Cool, where can I see how ubunutu’s is built? We may not have an issue at all (outside of patches).

**Comment by wsnipex** [comment]
https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/ffmpeg/7:4.2.2-1ubuntu1/ffmpeg_4.2.2-1ubuntu1.debian.tar.xz
check the debian/rules file

**Comment by phunkyfish** [comment]
Ha, ya, it covers all the bases and should be fine. I'll add a note to the readme stating it may be slightly behind due to patches but should generally have the same featureset as the one we build.

File: debian/control [hunk 1]
```diff
@@ -1,9 +1,10 @@
 Source: kodi-inputstream-ffmpegdirect
 Priority: extra
 Maintainer: Ross Nicholson <phunkyfish@gmail.com>
-Build-Depends: debhelper (>= 9.0.0), cmake,
-               kodi-addon-dev, pkg-config, libssl-dev, zlib1g-dev, yasm
-Standards-Version: 3.9.6
+Build-Depends: debhelper (>= 9.0.0), cmake, kodi-addon-dev,
+               libavutil-dev (>= 4.2.2), libavcodec-dev (>= 4.2.2), libavformat-dev (>= 4.2.2), libavfilter-dev (>= 4.2.2),
+               libbz2-dev, libp8-platform-dev, libssl-dev, libz3-dev, pkg-config, yasm, zlib1g-dev
+Standards-Version: 3.9.8
```
Comment (AlwinEsch) [inline debian/control:7]: Make it sense to use there also `4.1.2`? 
Mostly all addons have this currently and to hold a bit equal.

Gives it a table which version match witch ubuntu and debian?

Comment (wsnipex) [reply debian/control:7]: iirc 4.1.2 is debhelper 11, so we could use that, but it does not really make a difference atm

Comment (AlwinEsch) [reply debian/control:7]: Thought only to have them equal, that it works I seen by older addons.

Was also not sure how is in latest, but have looked now his debhelper changelog.
There is now `4.5.0` as debhelper 12 and except `4.1.4` (where `Beyond the dh_usrlocal change listed above, no changes were required.`) with only a `no changes were required` everywhere.

Therefore, you can leave it as it is or update it, regardless of the behavior of the versions there.

**Review by AlwinEsch — APPROVED** [review]

**Comment by phunkyfish** [comment]
Thanks @wsnipex

**Review by phunkyfish — COMMENTED** [review]
LGTM

**Pull request merged**